tests/Integration/ApiPlatform/ModuleEndpointTest.php

```diff
@@ -646,6 +651,90 @@ public function testBulkUninstallModule()
         }
     }
 
+    /**
+     * @depends testInstallModule
+     */
+    public function testUpgradeModule(): void
+    {
+
+        $module212 = array(
```

Comment:
```suggestion
        $module212 = [
```
This won't be accepte by the linter, you can run `./vendor/bin/php-cs-fixer fix` to fix all the errors